### ISSUE TITLE
conditional compilation depending on :periods-use-series feature is inconvenient

### ISSUE THREAD

**Issue by avodonosov** [jwiegley/periods#5]
Example use case.

When you `(ql:quickload :cl-ledger)` which depends on periods, cl-ledger pushes `:periods-use-series` into `*features*` so that periods is compiled with dependency on the `series` package.

When later, in another lisp session, you `(ql:quickload :periods)` the periods.fasl file is already exists in ~/.cache/common-lisp/.../. The .fasl file refers the  `series` package, but the package is not loaded by ASDF, as we don't push `:periods-use-series` into `*features*`. In result it the load operation fails.

In short, conditional compilation is inconvenient. It would be better, for example, to separate the series-dependent code into another ASDF system, so that we have two ASDF systems: periods.asd and periods+series.asd
